server/svix-server/src/static/docs.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Svix API - ReDoc</title>
+<!-- needed for adaptive design -->
+<meta charset="utf-8"/>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
+
+<link rel="shortcut icon" href="https://fastapi.tiangolo.com/img/favicon.png">
```

Comment:
This is the fastapi icon... :)
You can just remove this line for now, or just point to something on svix.com.